client/state/google-my-business/selectors.js

```diff
@@ -8,3 +8,6 @@ import { get } from 'lodash';
 export function getInterval( state, siteId, statType ) {
 	return get( state.googleMyBusiness, [ siteId, 'statInterval', statType ], 'week' );
 }
+
+export const getGoogleMyBusinessSiteStats = ( state, siteId, statType, interval, aggregation ) =>
+	get( state, [ 'googleMyBusiness', siteId, 'stats', statType, interval, aggregation ], null );
```

Comment:
We should move this selector to `/state/selectors` - that's where they should all be located.